presto-verifier/src/main/java/com/facebook/presto/verifier/prestoaction/JdbcPrestoAction.java

```diff
@@ -120,9 +129,11 @@ public QueryActionStats execute(Statement statement, QueryStage queryStage)
     {
         return prestoRetry.run(
                 "presto",
-                () -> networkRetry.run(
-                        "presto-cluster-connection",
-                        () -> executeOnce(statement, queryStage, statementExecutor)));
+                () -> throttleRetry.run(
+                        "request-throttle",
+                        () -> networkRetry.run(
+                                "presto-cluster-connection",
+                                () -> executeOnce(statement, queryStage, statementExecutor))));
```

Comment:
Having nested retry could drastically increase the max retry time because they're multiplicative. What about retry request throttling as part of the cluster connection retry?

Reply:
merged it into network retry